src/Native/Windows/CMakeLists.txt

```diff
@@ -91,14 +91,12 @@ endif()
 
 # App-Container builds require a more specific linking to ensure the APIs imported pass WACK
 if ($ENV{__appContainer} STREQUAL true AND UPPERCASE_CMAKE_BUILD_TYPE STREQUAL "RELEASE")
-    set(__LinkCustomCRT true)
     set(__LinkArgs "${__LinkArgs} /LTCG")
-endif()
-
-if (__LinkCustomCRT STREQUAL "true" AND $ENV{__BuildArch} STREQUAL "arm" )
-    set(__LinkArgs "${__LinkArgs} /DEFAULTLIB:ole32.lib")
-elseif(__LinkCustomCRT STREQUAL "true" AND NOT $ENV{__BuildArch} STREQUAL "arm" )
-    set(__LinkArgs "${__LinkArgs} /NODEFAULTLIB")
+    if ($ENV{__BuildArch} STREQUAL "arm" )
+        set(__LinkArgs "${__LinkArgs} /DEFAULTLIB:ole32.lib")
+    else()
+        set(__LinkArgs "${__LinkArgs} /NODEFAULTLIB")
```

Comment:
This `/NODEFAULTLIB` is likely the reason why it does not link. Try removing this line.

Reply:
+1. That's the issue. the crt.cpp file is providing the same thing as the default libs, so you need one or the other.